db/schema.rb

```diff
@@ -61,10 +53,13 @@
     t.string   "title"
     t.text     "body"
     t.integer  "price"
+    t.integer  "topic_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
 
+  add_index "sponsored_posts", ["topic_id"], name: "index_sponsored_posts_on_topic_id"
```

Comment:
You'll need to roll back your last migration using: `rake db:rollback` and re-run your migration again: `rake db:migrate` for your changes to the migration file to take place. You might also need to run `rake db:test:prepare`